src/Snapshot.php

```diff
@@ -88,49 +94,73 @@ protected function shouldIgnoreLine(string $line): bool
         return empty($line) || $this->isASqlComment($line);
     }
 
-    protected function loadStream(?string $connectionName = null): void
+    protected function loadStream(string $connectionName = null)
```

Comment:
```suggestion
    protected function loadStream(?string $connectionName = null): void
```

Please add the return type back, as well as the explicit `null` parameter type, as implicit nullability is deprecated since PHP 8.4.

Reply:
done